src/Module/Settings/Display.php

```diff
@@ -128,6 +131,8 @@ public static function content(array $parameters = [])
 			$default_mobile_theme = 'none';
 		}
 
+		$user = User::getById(local_user());
```

Comment:
What's the functional difference between `local_user()` and `App->getUserId()`?

Reply:
I'm using `App->getUserId()` when previously `App->user` had been used. And I'm using `local_user()` when in the code it is obvious that the `local_user()` is the right one. I'm currently unsure if these two values can differ at all. But I preferred to not open just another can of worms :-)

Reply:
`local_user()` comes from the SESSION superglobal, the `App->getUserId()` comes from the authentication method, we should only rely on the latter.